docker/vendor/nginx-es/Dockerfile

```
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM astronomerinc/ap-base:0.7.5
+FROM alpine:3.10.2
```

```suggestion
FROM alpine:3.10
```

So that it would update to minor alpine versions automatically? (I'm not clear what Alpine's versioning policy for the whole OS is.)

---

correct, but how can i trigger to rebuild, if in drone we have cached 3.10 tag?

---

Oh yes, fair point.